tests/Model/Request/RequestTest.php

```diff
@@ -285,4 +285,23 @@ public function testPersistingOfRuntimeChanges(): void
         self::assertArrayHasKey($fieldType, $queryParameters['fields']);
         self::assertStringContainsString($fieldName, $queryParameters['fields'][$fieldType]);
     }
+
+    public function testCustomQueryParameters(): void
+    {
+        $request = new Request(
+            'GET',
+            new Uri('/index.php/api/examples/example-1'),
```

Comment:
You should also add some protocol related query parameters to assure that they are still there after initialization.